newdle/client/src/components/DurationPicker.js

```diff
@@ -17,16 +32,14 @@ export default function DurationPicker({minDuration, maxDuration, interval}) {
   const dispatch = useDispatch();
 
   return (
-    <div className={styles['duration-picker']}>
+    <div>
       <span>Meeting time: </span>
```

Comment:
i think it might look better without the `:`

Reply:
Indeed.